docs/source/intro.rst

```diff
@@ -0,0 +1,75 @@
+.. _intro:
+
+Introduction: why is it complex to assess DEM accuracy and precision?
+=====================================================================
+
+Digital Elevation Models are numerical, gridded representations of elevation. They are generated from different instruments (e.g., optical sensors, radar, lidar), acquired in different conditions (e.g., ground, airborne, satellite), and using different post-processing techniques (e.g., photogrammetry, interferometry).
+
+While some complexities are specific to certain instruments and methods, all DEMs generally possess:
+
+- a `ground sampling distance <https://en.wikipedia.org/wiki/Ground_sample_distance>`_ (GSD), or pixel size, **that does not necessarily represent the underlying spatial resolution of the observations**,
+- a `georeferencing <https://en.wikipedia.org/wiki/Georeferencing>`_ **that can subject to shifts, tilts or other deformations** due to inherent instrument errors, noise, or associated post-processing schemes,
+- a large number of `outliers <https://en.wikipedia.org/wiki/Outlier>`_ **that remain difficult to filter** as they can originate from various sources (e.g., photogrammetric blunders, clouds).
+
+These factors lead to difficulties in assessing the accuracy and precision of DEMs, which are necessary to perform further analysis.
+
+In ``xdem``, we provide a framework with state-of-the-art methods published in the scientific literature to make DEM calculations consistent, reproducible, and easy.
+
+Accuracy and precision
+**********************
```

Comment:
`rST` is weird with section headers. Any order seems to work as long as you're consistent, but in some cases, that also doesn't seem to be the case!

The official order is:
-  \# with overline, for parts
-  \* with overline, for chapters
-  =, for sections
-  -, for subsections
- ^, for subsubsections
-  ", for paragraphs

I think it's best if we stay consistent in the documentation, so I recommend we use this order!

Reference:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Reply:
For this particular header, I think the recommended character would be:

```rst
Accuracy and precision
----------------------
```

Reply:
Changed with the official order!